src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -1636,4 +1636,16 @@ public void shouldFailOnSslExceptionWithLinkToTroubleShootingGuide() throws Inte
                   "https://docs.snowflake.com/en/user-guide/client-connectivity-troubleshooting/overview"));
     }
   }
+
+  /** Added in > 3.20.0 */
+  @Test
```

Comment:
Is this test to be verified by manual observation? Maybe we can check if a correct handler is used using something like:
```
rootLogger = ...
rootLogger.getHandlers().stream()
   .filter(handler -> handler instanceof ConsoleHandler)
   .get()
   .equals(expectedHandler)
```

Reply:
I agree with Piotrek. The main assertion should verify whether the StdoutConsoleHandler is used when the new flag is set and not used when we have a default configuration.

Reply:
I added assertion and moved the test to place where we can access new class